contracts/injectorFactoryV2.sol

```diff
@@ -53,4 +71,26 @@ contract ChildChainGaugeInjectorV2Factory {
     function getDeployedInjectors() external view returns (address[] memory) {
         return deployedInjectors;
     }
+
+    /**
+     * @notice Registers an injector in the Chainlink Keeper Registry
+     * @param _injector The address of the injector to be registered
+     */
+    function _registerUpkeep(address _injector, address _owner) internal returns (uint256 upkeepId_) {
+        IKeeperRegistrar.RegistrationParams memory registrationParams = IKeeperRegistrar.RegistrationParams({
+            name: "ChildChainGaugeInjectorV2",
+            encryptedEmail: "",
+            upkeepContract: _injector,
+            gasLimit: 2_000_000,
+            adminAddress: _owner,
+            triggerType: 0,
+            checkData: "",
+            triggerConfig: "",
+            offchainConfig: "",
+            amount: 1e18
+        });
+
+        upkeepId_ = CL_REGISTRAR.registerUpkeep(registrationParams);
```

Comment:
Need 1(amount) $LINK for this to work.  Suggest expecting the user to approve link to the helper, and then transferFrom(msg.sender, address(this), 1e18) before making this call.